config/config.yaml

```diff
@@ -14,7 +14,7 @@ reference:
   design_bed: "pgx/padd_target_regions/padded_bait_interval.bed"
   fasta: "/mnt/WD1/ref/Uppsala/Data/BWA_0.7.10_refseq/hg19.with.mt.fasta"
   fai: "/mnt/WD1/ref/Uppsala/Data/BWA_0.7.10_refseq/hg19.with.mt.fai"
-#  dbsnp: "/mnt/WD1/ref/dbsnp/dbSNP_153/hg19/00-All.vcf.gz"
+  #dbsnp: "/mnt/WD1/ref/dbsnp/dbSNP_153/hg19/00-All.vcf.gz"
```

Comment:
```suggestion
```

remove commented settings

Reply:
Right, thanks. I forgot but I don't need this anymore, so I'll delete that line.